docs/content/theme/color-palette.md

```diff
@@ -18,7 +18,11 @@ tags = [ "theme" ]
 
 ## Text & links
 
-Text and links are the most fundamental pieces of any web property, so they have their own variables aside from other UI elements. They also respond to context, such as a bit of text in a dark gray card would need to change to white. Or a link in a dark gray card would need to become a lighter shade of blue.  Here are a few examples:
+Text and links are the most fundamental pieces of any web property, so they have their own variables aside from other UI elements. They also respond to context, such as a bit of text in a dark gray card would need to change to white. Or a link in a dark gray card would need to become a lighter shade of blue.  
+
+You may choose to set the value of a link color to something in your brand, but it is not required, and these colors should provide a good user experience. 
```

Comment:
```suggestion
You can set the value of these typography color variables to something in your brand palette but be sure to validate them against a [contrast checker](https://webaim.org/resources/contrastchecker/) to ensure they continue to pass accessibility standards when set against your surface or UI palettes.  Aim for AAA status if you can! ⭐ 
```

Reply:
I thought this added a little more guidance about how the variables relate to the palette and how to validate your updates are okay.